src/org/omegat/core/data/PluginInformation.java

```diff
@@ -173,4 +267,55 @@ public boolean equals(Object obj) {
         return true;
     }
 
+    @Override
+    public final int compareTo(PluginInformation pluginInformation) {
```

Comment:
Here is a point how to sort plugins, that affect to order of preferences-plugins table in default.